app/models/task.rb

```diff
@@ -6,4 +6,18 @@ class Task < ApplicationRecord
   before_destroy { |record| raise ActiveRecord::ReadOnlyRecord if record.readonly? }
 
   delegate :readonly?, to: :story
+
+  def to_csv
+    [name, status]
+  end
+
+  private
+
+  def status
+    if done
+      'completed'
+    else
+      'not_completed'
```

Comment:
@hjJunior It looks like this branch of code isn't tested. I think we should add a spec for when `done` is false.

Reply:
I'll work to add this spec, thanks